libs/blocks/global-navigation/utilities/menu/menu.css

```diff
@@ -129,6 +137,14 @@
     margin-right: 0;
   }
 
+  .feds-menu-column > ul {
+    padding: 0;
+  }
+
+  .feds-menu-section > a:first-child {
+    margin: 0;
```

Comment:
Maybe we could cancel out exactly what we add in the first place, which is just the `margin-top`

Reply:
Done 👍